drv/distrobox/distrobox.go

```diff
@@ -45,6 +46,14 @@ func (up DistroboxUpdater) New(config UpdaterInitConfiguration) (DistroboxUpdate
 
 	up.binaryPath = EnvOrFallback(up.Config.Environment, "UUPD_DISTROBOX_BINARY", "/usr/bin/distrobox")
 
+	inf, err := os.Stat(up.binaryPath)
+	if err != nil {
+		up.Config.Enabled = false
+		return up, err
+	}
+	// check if file is executable using bitmask
+	up.Config.Enabled = inf.Mode()&0111 != 0
```

Comment:
We also need to check if the other ones exit when they get executed. Like, on that `BootcCompatible` function and stuff. I'd also remove `RunLog` from those.

Reply:
the bootc compat already checks if it exits. IMO we don't need these checks for flatpak or the system because they're expected to be there, and if they aren't there, the program *should* throw an unexpected error

We also don't use RunLog here, just a CombinedOutput:
```go
func BootcCompatible(binaryPath string) (bool, error) {
	cmd := exec.Command(binaryPath, "status", "--format=json")
	out, err := cmd.CombinedOutput()
	if err != nil {
		return false, nil
	}
	var status bootcStatus
	err = json.Unmarshal(out, &status)
	if err != nil {
		return false, nil
	}
	return !(status.Status.Booted.Incompatible || status.Status.Staged.Incompatible), nil
}
```